platform/mds/mds-web/src/main/resources/webapp/js/directives.js

```diff
@@ -2268,9 +2268,7 @@
                         fieldPath = fieldPath.substring(fieldPath.indexOf('.') + 1);
                     }
 
-                    value = _.isBoolean(ngModel.$modelValue)
-                        ? !ngModel.$modelValue
-                        : ngModel.$modelValue;
+                    value = ngModel.$modelValue;
```

Comment:
@pgesek I checked it again and found that not only settings but also validations are not being saved (not only boolean values). So, I'll update the description and also rework on this ticket.